frontend/packages/pipelines-plugin/src/components/pipelines/modals/triggers/remove-utils.ts

```diff
@@ -41,7 +41,7 @@ export const removeTrigger = async (values: RemoveTriggerFormValues, pipeline: P
     updatableEventListeners.map((eventListener) =>
       k8sPatch(EventListenerModel, eventListener, [
         {
-          opt: 'replace',
+          op: 'replace',
```

Comment:
looks like a bug

Reply:
@andrewballantyne Can you help verify this change?

I'm not sure what the default is if `op` is missing. `add` might actually be better since it handles the case where `spec.triggers` is missing, but will replace it if present.

Reply:
I think `spec.triggers` is a required field. (cc @karthikjeeyar) but yeah, this looks okay, still removes the trigger 🙂  Not sure how this was missed before. Guess if it works people don't look too closely lol.